nova_vm/src/ecmascript/builtins/text_processing/string_objects/string_prototype.rs

```diff
@@ -1478,3 +1513,44 @@ enum TrimWhere {
     End,
     StartAndEnd,
 }
+
+enum NormalizeForm {
+    NFC,
```

Comment:
**issue:** Apparently clippy doesn't like these names and wants these to be `Nfc`, `Nfd` etc. Let's give unto clippy what is theirs and do it that way.

Reply:
That's right. Just renamed them ;)